logstash-core/spec/logstash/plugin_spec.rb

```diff
@@ -377,6 +387,22 @@ def register; end
     end
   end
 
+  describe "deprecation logger" do
+    let(:config) do
+      {
+        "host" => "127.0.0.1"
+      }
+    end
+
+    context "when a plugin is registered" do
+      subject { CustomFilterDeprecable.new(config) }
+
+      it "return a human readable ID" do
```

Comment:
this description `it "return a human readable ID"`  does not really express what the assertion does: `expect(subject.deprecation_logger).not_to be_nil` which only checks it is not `nil`, not that is it human readable?